chef/site-cookbooks/wca/templates/wca_https.conf.erb

```diff
@@ -1,15 +1,56 @@
+### 
+# Following this :
+# https://raymii.org/s/tutorials/Strong_SSL_Security_On_nginx.html
```

Comment:
This guide looks good, although I usually use https://mozilla.github.io/server-side-tls/ssl-config-generator/ (which stays up to date).